src/reponseHandlers.js

```diff
@@ -0,0 +1,29 @@
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function sendErrorResponse(response, errors) {
+  response.statusCode = 400;
+  response.statusMessage = 'Bad request';
```

Comment:
I would create some kind of config file, which has all codes and messages, and then use it throughout the project
```javascript
const BAD_REQUEST = {
  code: 400,
  message: 'Bad request'
};
```